cmd/service.go

```diff
@@ -0,0 +1,190 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/redhat-developer/odo/pkg/application"
+	"github.com/redhat-developer/odo/pkg/project"
+	svc "github.com/redhat-developer/odo/pkg/service"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var serviceForceDeleteFlag bool
+
+// serviceCmd represents the service command
+var serviceCmd = &cobra.Command{
+	Use:   "service",
+	Short: "Perform service catalog operations",
+	Long: ` Perform service catalog operations (Currently we are limiting to template service broker only
+because, Ansible Service Broker is highly parameterised, whereas Template Service broker comes with templates which has default values )`,
```

Comment:
Long: ` Perform service catalog operations.

Limited to template service broker only. )`,

May be better. The long description seems a bit *too* long. No need to explain *why* we don't support it.